content/building/yaml.md

```diff
@@ -77,9 +77,10 @@ This is the skeleton structure of `codemagic.yaml`.
           vars:
             PUBLIC_ENV_VAR: "value here"
           flutter: stable
+          xcode: latest
         cache:
           cache_paths:
-            - $FCI_BUILD_DIR/build
+            - ~/.pub
```

Comment:
Please add `~.pub-cache` instead of `.pub`.

Reply:
we can't cache `$FCI_BUILD_DIR/build` since it causes compilation errors

Reply:
```suggestion
            - ~/.pub-cache
```